suite-native/config/src/supportedNetworks.ts

```diff
@@ -3,7 +3,7 @@ import { A } from '@mobily/ts-belt';
 import { isTestnet } from '@suite-common/wallet-utils';
 import { Network, NetworkSymbol, getMainnets, getTestnets } from '@suite-common/wallet-config';
 
-const discoveryBlacklist: NetworkSymbol[] = ['sol', 'dsol'];
+const discoveryBlacklist: NetworkSymbol[] = ['sol', 'dsol', 'matic'];
```

Comment:
shoudn't we blacklist it also for imports?

Reply:
In portfolio tracker, it's not in the whitelist (which is used to offer coins to be imported) and since it cannot be imported, there's no need to blacklist it. We just needed to blacklist it for the device discovery, since device supports matic, without this blacklist the coin gets discovered and added as account.

Reply:
(Video shows it's not offered in the coins that can be imported)